test/simulator/test/org/apache/cassandra/simulator/test/ShortPaxosSimulationTest.java

```diff
@@ -114,7 +112,9 @@ public void selfReconcileTest()
                                                                           "-c", "2",
                                                                           "--cluster-action-limit", "2",
                                                                           "-s", "30",
-                                                                          "--with-self" });
+                                                                          "--with-self",
+                                                                          "--with-rng", "0",
```

Comment:
why this change?

Reply:
It enables additional validation points where threads wait to check the simulation hasn't diverged. Specifically time and random number generation.